### PR TITLE
feat: Add FromDateToPrettyStringConverter to WinUI

### PR DESCRIPTION
**Pull request by lesterbotello**
GitHub Issue: #

## Proposed Changes
Adds the `FromDateToPrettyStringConverter` to the `Uno.WinUI` project.

- Feature

## What is the current behavior?
`FromDateToPrettyStringConverter` is not available in the WinUI project

## What is the new behavior?
`FromDateToPrettyStringConverter` is available in the WinUI project

## Checklist

Please check if your PR fulfills the following requirements:

- [ ] ~~Documentation has been added/updated~~
- [ ] ~~Automated Unit / Integration tests for the changes have been added/updated~~
- [X] Contains **NO** breaking changes
- [ ] ~~Updated the Changelog~~
- [ ] ~~Associated with an issue~~

### CODE REVIEW

File: src/View.Uno.WinUI/View.Uno.WinUI.csproj [hunk 1]
```diff
@@ -75,10 +75,115 @@
...
+	<PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Release|netstandard2.0|AnyCPU'">
+	  <DefineConstants>$(DefineConstants);__WASM__;__NETSTANDARD__</DefineConstants>
+	</PropertyGroup>
+
 	<ItemGroup>
 		<UpToDateCheckInput Include="**\*.xaml" Exclude="bin\**\*.xaml;obj\**\*.xaml" />
 	</ItemGroup>
 
+	<ItemGroup>
+	  <PRIResource Remove="Converters\PrettyDateConverter\Resources\de\PrettyDateFormatterStrings.resw" />
```
Comment (jeanplevesque) [inline src/View.Uno.WinUI/View.Uno.WinUI.csproj]: Could we reduce those `PRIResource Remove` to 1 line by using a globe pattern?

Comment (jeanplevesque) [reply src/View.Uno.WinUI/View.Uno.WinUI.csproj]: And actually, the same comment applies for the `None Include` and the `EmbededResource`.

File: src/View.Uno.WinUI/Converters/PrettyDateConverter/PrettyDateFormatter.cs [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+		}
+
+		public string Format(TimeSpan elapsed, CultureInfo culture, PrettyDateMode mode = PrettyDateMode.Past)
+		{
+			if (elapsed.TotalSeconds < 1)
+			{
+				elapsed = TimeSpan.Zero;
+			}
+
+			//if (dateTime.Age(now) >= 1)
```
Comment (Soap-141) [inline src/View.Uno.WinUI/Converters/PrettyDateConverter/PrettyDateFormatter.cs:40]: Why is this commented out?

**Review by Soap-141 — DISMISSED** [review]

File: src/View.Uno.WinUI/View.Uno.WinUI.csproj (hunk 1 above)
Comment (lesterbotello) [reply src/View.Uno.WinUI/View.Uno.WinUI.csproj]: Done.

File: src/View.Uno.WinUI/Converters/PrettyDateConverter/PrettyDateFormatter.cs (hunk 2 above)
Comment (lesterbotello) [reply src/View.Uno.WinUI/Converters/PrettyDateConverter/PrettyDateFormatter.cs:40]: Can't say... This is a verbatim copy of the UWP implementation.

**Review by Soap-141 — APPROVED** [review]

**Pull request merged**